Lib/subprocess.py

```diff
@@ -1414,6 +1419,16 @@ def _execute_child(self, args, executable, preexec_fn, close_fds,
 
             if executable is None:
                 executable = args[0]
+
+            if (preexec_fn is None
+                    and not close_fds
+                    and not pass_fds
+                    and cwd is None
+                    and env is None
+                    and restore_signals is False
+                    and not start_new_session):
+                return self._posix_spawn(args, executable)
```

Comment:
I suggest to pass (executable, args) to follow os.posix_spawn() API. The function has not return value, so I suggest to write instead:

```suggestion
                 self._posix_spawn(executable, args)
                 return
```